### PR TITLE
doc: added rbd default features

### PR DESCRIPTION
**Pull request by gauravsitlani**
Added RBD default features along with the description.

Fixes:https://tracker.ceph.com/issues/15000
Signed-off-by: Gaurav Sitlani gauravsitlani@riseup.net

- [https://tracker.ceph.com/issues/15000] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by dillaman** [comment]
jenkins render docs

File: doc/rbd/rbd-config-ref.rst [hunk 1]
```diff
@@ -134,3 +134,54 @@ Read-ahead is automatically disabled if caching is disabled.
 :Type: 64-bit Integer
 :Required: No
 :Default: ``50 MiB``
+
+
+RBD Default Features
+========================
```
Comment (dillaman) [inline doc/rbd/rbd-config-ref.rst]: Nit: the underlining should match in length to its text

File: doc/rbd/rbd-config-ref.rst [hunk 2]
```diff
@@ -134,3 +134,54 @@ Read-ahead is automatically disabled if caching is disabled.
...
+
+RBD supports these features which can be specified via the command line when creating images or the default features can be specified via Ceph config file via 'rbd_default_features = <sum of feature numeric values>' or 'rbd_default_features = <comma-delimited list of CLI feature names>'
```
Comment (dillaman) [inline doc/rbd/rbd-config-ref.rst]:
```suggestion
RBD supports advanced features which can be specified via the command line when creating images or the default features can be specified via Ceph config file via 'rbd_default_features = <sum of feature numeric values>' or 'rbd_default_features = <comma-delimited list of CLI values>'
```

File: doc/rbd/rbd-config-ref.rst [hunk 3]
```diff
@@ -134,3 +134,54 @@ Read-ahead is automatically disabled if caching is disabled.
...
+
+``Layering``
+
+:Description: Layering enables you to use cloning.
+:Config numeric value: 1
```
Comment (dillaman) [inline doc/rbd/rbd-config-ref.rst]: I'm not familiar enough w/ rst, but for some reason the value is getting moved down to the next line [1]

[1] http://docs.ceph.com/ceph-prs/24720/rbd/rbd-config-ref/#rbd-default-features

Comment (gauravsitlani) [reply doc/rbd/rbd-config-ref.rst]: I'll search if this can be resolved.

File: doc/rbd/rbd-config-ref.rst (hunk 1 above)
Comment (gauravsitlani) [reply doc/rbd/rbd-config-ref.rst]: Sure i'll modify it.

**Comment by joscollin** [comment]
@gauravsitlani Squash the commits https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by karunjosy** [comment]
jenkins render docs

File: doc/rbd/rbd-config-ref.rst (hunk 3 above)
Comment (gauravsitlani) [reply doc/rbd/rbd-config-ref.rst]: Can we shorten this 'Config numeric value' keyword , its long as compared to the other ones . I think that's why the doc is not getting rendered properly.

Comment (dillaman) [reply doc/rbd/rbd-config-ref.rst]: Sure -- perhaps try ```Internal value```?

File: doc/rbd/rbd-config-ref.rst [hunk 4]
```diff
@@ -134,3 +134,54 @@ Read-ahead is automatically disabled if caching is disabled.
 :Type: 64-bit Integer
 :Required: No
 :Default: ``50 MiB``
+
+
+RBD Default Features
+=====================
```
Comment (dillaman) [inline doc/rbd/rbd-config-ref.rst]: Nit: still looks one character too long

**Comment by gauravsitlani** [comment]
jenkins render docs

**Comment by gauravsitlani** [comment]
> Doc render available at http://docs.ceph.com/ceph-prs/24720/
@dillaman Its getting rendered properly now. One thing i observe here is when the Description exceeds one line the keyword 'Internal Value' breaks to a new line. Is it fine or do we need some change?

**Comment by dillaman** [comment]
@gauravsitlani My only remaining comment is that it would be nice if you could squash your 4 commits down to a single commit. Thx.

**Pull request merged**